doc/Extensions/Applications.md

```diff
@@ -539,7 +539,7 @@ extend postgres /etc/snmp/postgres
 
 6: Verify the path to check_postgres.pl in /etc/snmp/postgres is correct.
 
-7: If you wish it to ignore the database postres for totalling up the stats, set ignorePG to 1(the default) in /etc/snmp/postgres. If you are using netdata or the like, you may wish to set this or otherwise that total will be very skewed on systems with light or moderate usage.
+7: If you wish it to ignore the database postges for totalling up the stats, set ignorePG to 1(the default) in /etc/snmp/postgres. If you are using netdata or the like, you may wish to set this or otherwise that total will be very skewed on systems with light or moderate usage.
```

Comment:
Hah! Was just about to reply with saying it is suppose to be lower case when I noticed the missing r.